bvm/ballerina-runtime/src/main/java/org/ballerinalang/jvm/values/ErrorValue.java

```diff
@@ -67,7 +69,16 @@ public ErrorValue(BType type, String reason, Object details) {
 
     @Override
     public String stringValue() {
-        return "error " + reason + Optional.ofNullable(details).map(details -> " " + details).orElse("");
+        BType type = TypeChecker.getType(details);
+        return "error " + reason + Optional.ofNullable(details).map(details -> " " + StringUtils.getStringValue(null,
```

Comment:
This can be replaced with return stringValue(null);

Reply:
fixed